pagerduty/service.go

```diff
@@ -85,6 +85,31 @@ type Service struct {
 	Type                   string                     `json:"type,omitempty"`
 }
 
+// ServiceEventRule represents a service event rule
+type ServiceEventRule struct {
+	ID         string                      `json:"id,omitempty"`
+	Self       string                      `json:"self,omitempty"`
+	Disabled   bool                        `json:"disabled,omitempty"`
+	Conditions *RuleConditions             `json:"conditions,omitempty"`
+	TimeFrame  []*RuleTimeFrame            `json:"time_frame,omitempty"`
+	Variables  []*ServiceEventRuleVariable `json:"variables,omitempty"`
+	Position   int                         `json:"position,omitempty"`
+	Actions    *RuleActions                `json:"actions,omitempty"`
```

Comment:
The Service Event Rule Actions are a little different from the Ruleset Rule Actions. The main one for this library is there is no `route` action on Service Event Rules. Also, I'm told there will be new actions coming, so it may be smarter to create it's own type for `ServiceEventRuleAction` or something like that.